frontend/public/components/overview/_project-overview.scss

```diff
@@ -160,6 +160,14 @@
   }
 }
 
+.project-overview__status .co-icon-and-text {
+  margin-right: 12px;
+
+   .project-overview__builds & {
+    margin-right: 4px;
```

Comment:
indentation :)

Reply:
Thanks.  Fixed.  That's what happens when I put up a PR while trying to simultaneously rushing out the door.